pkgs/data/themes/materia-theme-transparent/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  fetchFromGitHub,
+  materia-theme,
+}:
+materia-theme.overrideAttrs (oldAttrs: rec {
+  pname = "materia-theme-transparent";
+  version = "20210322";
```

Comment:
```suggestion
  version = "0-unstable-2021-03-22";
```

Reply:
Just wondering, what does the prefix exactly mean? I get changing the date (that was just copied from the upstream nixpkgs materia-theme). Thanks for the suggestions.

Reply:
This is just the nixpkgs versioning scheme for packages without a proper release. Since there haven't been any releases, we prefix the [0 version](https://0ver.org/).